src/mantarray_desktop_app/main_process/process_monitor.py

```diff
@@ -225,7 +225,10 @@ def _check_and_handle_server_to_main_queue(self) -> None:
             else:
                 raise NotImplementedError(f"Unrecognized shutdown command from Server: {command}")
         elif communication_type == "update_user_settings":
-            new_values = communication["content"]
+            # if content does not exist then check config_settings
+            new_values = (
+                communication["content"] if "content" in communication else communication["config_settings"]
```

Comment:
When does this get sent as `communication["config_settings"]`? 
https://github.com/CuriBio/mantarray-desktop-app/blob/def1d998ae422d4f5915e11271132e3d64e8a4bd/src/mantarray_desktop_app/main_process/server.py#L428